app/src/main/java/com/framgia/fsalon/screen/stylistbooking/StylistBookingPresenter.java

```diff
@@ -24,6 +29,11 @@
  */
 public class StylistBookingPresenter implements StylistBookingContract.Presenter {
     private static final String TAG = StylistBookingPresenter.class.getName();
+    private static final String STATUS = String.valueOf(STATUS_CANCELED)
```

Comment:
WARNING: [UselessStringValueOf] No need to call String.valueOf to append to a string.
https://pmd.github.io/pmd-5.5.1/pmd-java/rules/java/strings.html#UselessStringValueOf